ts/nni_manager/training_service/reusable/environments/amlEnvironmentService.ts

```diff
@@ -91,13 +91,16 @@ export class AMLEnvironmentService extends EnvironmentService {
                 case 'COMPLETED':
                 case 'SUCCEEDED':
                     environment.setStatus('SUCCEEDED');
+                    this.stopEnvironment(environment);
```

Comment:
Seems the `stopEnvironment()` function is called in trialDispatcher.ts to clean up environment, if this function is not called appropriate, suggest to fix logic in trialDispatcher instead of here, for other platforms might has this issue too.

Reply:
Good job, dude, I meet the same problem, it works!